src/modules/importer.js

```diff
@@ -0,0 +1,120 @@
+import fs from 'fs';
+
+export default class Importer {
+  listen(eventEmitter, path, sync = false) {
+    this.eventEmitter = eventEmitter;
+
+    this.eventEmitter.on('changed', () => {
+      if (sync) {
+        console.log(this.importSync(path));
+      } else {
+        let result = this.import(path);
+
+        result.then(data => {
+          console.log(data);
+        });
+      }
+    });
+  }
+
+  import(path) {
+    return new Promise((resolve, reject) => {
+      fs.readdir(path, (err, files) => {
+        if (err) {
+          return reject(err);
+        }
+
+        let content = this.readFiles(path, files);
+        content.then(csv => {
+          return this.csv(csv);
+        }).then(data => {
+          resolve(data);
+        });
+      });
+    });
+  }
+
+  importSync(path) {
+    let files = fs.readdirSync(path);
+    let csv = this.readFilesSync(path, files);
+    let content = this.csvSync(csv);
+
+    return content;
+  }
+
+  readFiles(path, files) {
+    let content = files.map(file => {
+      let fullPath = path + file;
+
+      return this.readFile(fullPath);
+    });
+
+    return Promise.all(content);
+  }
+
+  readFilesSync(path, files) {
+    let content = files.map(file => {
+      let fullPath = path + file;
+
+      return this.readFileSync(fullPath);
+    });
+
+    return content;
+  }
+
+  readFile(path) {
+    return new Promise((resolve, reject) => {
+      fs.readFile(path, 'utf8', (err, stats) => {
+        if (err) {
+          return reject(err);
+        }
+
+        resolve(stats);
+      });
+    });
+  }
+
+  readFileSync(path) {
+    return fs.readFileSync(path, 'utf8');

```

Comment:
utf-8 is by default - please remove this method

Reply:
Unfortunately, it returns raw buffer by default. Pls, correct me, if I wrong